pkg/autoscaler/scaling/multiscaler.go

```diff
@@ -219,8 +217,6 @@ func (m *MultiScaler) Get(_ context.Context, namespace, name string) (*Decider,
 // Create instantiates the desired Decider.
 func (m *MultiScaler) Create(ctx context.Context, decider *Decider) (*Decider, error) {
 	key := types.NamespacedName{Namespace: decider.Namespace, Name: decider.Name}
-	logger := m.logger.With(zap.String(logkey.Key, key.String()))
-	ctx = logging.WithLogger(ctx, logger)
```

Comment:
I believe this will mean logstream won't associate autoscaler records with the test?  Is this already done somewhere?  Worth double checking.

Reply:
AFAIK, `Create` is only called from the reconciler and thus `ctx` carries the correct logger.